src/AddressSpace.cc

```diff
@@ -2101,4 +2103,41 @@ bool AddressSpace::is_stap_semaphore(remote_ptr<uint16_t> addr) {
   return stap_semaphores.find(addr) != stap_semaphores.end();
 }
 
+void AddressSpace::update_fdt_uniform() {
```

Comment:
expand 'fdt' here please

Reply:
and add a comment explaining what this function does

Reply:
I've renamed this function to `fd_tables_changed`, which indicates when it should be called, rather than what it does.